api/src/main/java/org/pmiops/workbench/google/DirectoryService.java

```diff
@@ -6,8 +6,21 @@
 public interface DirectoryService {
```

Comment:
Would renaming this to `GSuiteDirectoryService` or similar be clearer at a glance? If you're looking in the tree view it's probably clear, but I like to jump around.